radsat-sk/operation/subsystems/RPdb.c

```diff
@@ -8,6 +8,7 @@
 #include <RI2c.h>
 #include <string.h>
 #include <RCommon.h>
+#include <RErrorManager.h>
```

Comment:
The Incude can go into RCommon just like RDebug.h and then you don't Need to have it in every file.